configs/beit/upernet_beit_base_640x640_160k_ade20k_ms.py

```diff
@@ -0,0 +1,67 @@
+_base_ = [
+    '../_base_/models/upernet_beit.py', '../_base_/datasets/ade20k_640x640.py',
+    '../_base_/default_runtime.py', '../_base_/schedules/schedule_160k.py'
+]
```

Comment:
might inherit configs/beit/upernet_beit_base_640x640_160k_ade20k.py and revise little